site-manage/site-group-manager/src/main/webapp/WEB-INF/templates/auto_groups_step1.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/common :: head" />
+<body>
+  <div th:replace="fragments/common :: jQuery" />
+  <div class="portletBody">
+    <div id="menu" th:include="fragments/menus :: main (autogroups)" />
+    <div class="page-header"><h1 th:text="#{autogroups.header.wizard}"></h1></div>
+    <div th:replace="fragments/wizard :: step (step1)" />
+    <div class="row" th:if="${step1ErrorMessage != null}">
+      <div class="col-sm-8">
+        <div class="sak-banner-error" th:text="${step1ErrorMessage}">You must at least select a role.</div>
+      </div>
+    </div>    
+    <div class="row" th:if="${!siteRoleList.isEmpty()}">
+      <div class="col-sm-8">
+        <div class="instruction" th:text="#{autogroups.step1.info}">Let's start with the first step. Please select which site role(s) to include in the groups you are creating.</div>
+      </div>
+    </div>
+    <div class="row" th:if="${!siteRoleList.isEmpty()}">
+      <div class="col-sm-8">
+          <form id="autogroups-wizard-step1-form" th:action="@{/autogroups/submitStep1}" th:object="${autoGroupsForm}" method="post">
+            <table class="table table-bordered table-striped table-hover">
+            <thead>
+              <tr>
+                <th><input type="checkbox" id="selectAllRoles" th:text="| #{autogroups.step1.table.selectallnone}|"/></th>
+                <th th:text="#{autogroups.step1.table.role}"></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr th:each="role : ${siteRoleList}">
+                <td><input type="checkbox" name="selectedRoleList" th:field="*{selectedRoleList}" th:value="${role.id}"/></td>
+                <td th:text="${role.id}"></td>
+              </tr>
+            </tbody>
+            </table>
+            <div class="act">
+              <input accesskey="x" type="submit" class="active" id="autogroups-continue-button" th:value="#{autogroups.button.continue}" onclick="SPNR.disableControlsAndSpin( this, null );">
+              <a th:href="@{/}" accesskey="x" id="autogroups-cancel-button" class="btn btn-default" th:text="#{autogroups.button.cancel}">Cancel</a>
+            </div>
+          </form>
+      </div>
+    </div>
+  </div>
+  <script th:inline="javascript">
```

Comment:
This should really be in a JS file. We want to move away from embedding scripts in pages.

Reply:
I'll use fragments instead.